.editorconfig

```diff
@@ -16,6 +16,3 @@ indent_size = 2
 
 [phars.xml]
 indent_size = 2
-
-[*.twig]
-insert_final_newline = false
```

Comment:
There was some file that needed this, but I can't remember which one.

Reply:
Yeah whitespace changes in templates can affect the display, so I wouldn't auto add/remove it.

Reply:
@markstory Can you revert this please.